src/ontology/mondo-edit.obo

```diff
@@ -454056,6 +454435,7 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6296" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6651" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6878" xsd:anyURI
```

Comment:
same as above, ticket annotation added but now new `is_a`

Reply:
Same. For what I can see, the annotation already existed (same is_a, same source) and was not added again.

Reply:
I gather this is also ok. As mentioned, in the first commit of the work (the one at this hash https://github.com/monarch-initiative/mondo/pull/6887/commits/2ec777ab79bcd005fb6de86ce9502e1223b893da) I look for pairs of changes to a class. E.g. did it receive both a new parent (`is_a`) and an annotation to reference the github ticket.